### PR TITLE
#76596 update icu version for norwegian culture

### LINKED ISSUE #76596: A DateTime.ToString with CultureInfo("nb-NO") gives the wrong datetime string output on iOS
_From @jfversluis on Mon, 03 Oct 2022 14:31:01 GMT_

---
**Issue moved from dotnet/maui#10441**
- Please respond to @sith-ikjetil.

---

_From @sith-ikjetil on Saturday, October 1, 2022 2:39:32 PM_

### Description

If you compile a .NET MAUI project for iOS, you get the wrong datetime format string for Norwegian output.
If you compile a .NET MAUI project for Windows or Android, you get the correct datetime format string for Norwegian output.

I use dotnet --version 6.0.401

### Steps to Reproduce

1. Create a new .NET MAUI APP and compile for iOS.
2. Anywhere like in 'MauiProgram.cs' do something like:
3. var dt = DateTime.Now.ToString( new CultureInfo("nb-NO"));
4. dt will look like: 10/1/2022 4:00:28 PM
5. dt should look like: 01.10.2022 16:00:286

### Link to public reproduction project repository

https://github.com/sith-ikjetil/maui-cultureinfo-no-bug

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15.4

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

_Copied from original issue xamarin/xamarin-macios#16217_

### PR DESCRIPTION
**Pull request by mkhamoyan**
Updating icu version after adding "nb-NO" culture in icu filters https://github.com/dotnet/icu/pull/271
Fix #76596

### CODE REVIEW

**Comment by steveisok** [comment]
@mkhamoyan is there a test in runtime which validates the culture is there?

**Comment by mkhamoyan** [comment]
> @mkhamoyan is there a test in runtime which validates the culture is there?

No, I modified and checked locally only. Will add test case checking that.

File: src/libraries/System.Globalization/tests/CultureInfo/CultureInfoAll.cs [hunk 1]
```diff
@@ -822,5 +822,15 @@ public void CultureNotFoundExceptionTest()
             e = AssertExtensions.Throws<CultureNotFoundException>("culture", () => new CultureInfo(0x1000));
             Assert.Equal(0x1000, e.InvalidCultureId);
         }
+
+        [Theory]
+        [PlatformSpecific(TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)] // for these platforms cultures are taken from icu filters 
+        [InlineData("nb-NO")]
+        public void ContainsCulture(string culture)
+        {
+			var cultures = CultureInfo.GetCultures(CultureTypes.AllCultures).ToList();
```
Comment (tarekgh) [inline src/libraries/System.Globalization/tests/CultureInfo/CultureInfoAll.cs:831]: Do you need `ToList()` here?

**Review by tarekgh — APPROVED** [review]

File: src/libraries/System.Globalization/tests/CultureInfo/CultureInfoAll.cs (hunk 1 above)
Comment (mkhamoyan) [reply src/libraries/System.Globalization/tests/CultureInfo/CultureInfoAll.cs]: Removed it.

**Comment by mkhamoyan** [comment]
Failures are unrelated to the changes.

**Pull request merged**